nntrainer/tensor/cl_operations/blas_kernel_interface.h

```diff
@@ -64,11 +64,10 @@ void multiplyCl(Tensor &input, float const &value);
 
 /**
  * @brief Process data and dimensions for add operation
- * @param[in] input Tensor
- * @param[in] result Tensor
- * @param[in] RunLayerContext reference
+ * @param[in] inputA Tensor
+ * @param[in] inputB Tensor
  */
-void add_i_cl(Tensor const &input, Tensor &result);
+void add_i_cl(Tensor &inputA, Tensor const &inputB);
```

Comment:
I'm assuming inputA is a result, and inputB is an input. is this correct?

Reply:
Both are inputs, the addition is taking inplace and inputA is returned as output